cosmo_tester/framework/cluster.py

```diff
@@ -213,9 +213,13 @@ def verify_services_are_running(self):
         status = self.client.manager.get_status()
         for service in status['services']:
             for instance in service['instances']:
-                assert instance['SubState'] == 'running', \
-                    'service {0} is in {1} state'.format(
-                            service['display_name'], instance['SubState'])
+                if (
+                    service['display_name'] == 'Cloudify Stage'
```

Comment:
Hey, reminding users that they're missing out on premium features is a marketing thing! ;P

I hope we'll never change the id, as it's used in many many places (since it's the unit name in systemd). However, this is the only place that depends on the display name, hence my comment.